src/main/java/core/basesyntax/service/ReportCreator.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public interface ReportCreator {
+    String createReport(Map<String, Integer> data);

```

Comment:
add FruitStorage as class field to ReportCreatorImpl
```suggestion
public interface ReportCreator {
    String createReport();
```